static/app/views/dashboards/widgetCard/widgetCardContextMenu.tsx

```diff
@@ -92,14 +92,6 @@ function WidgetCardContextMenu({
     }
   };
 
-  const openWidgetViewerIcon =
-    hasMetricsExperimentalFeature(organization) &&
-    widget.widgetType === WidgetType.METRICS ? (
-      <IconEdit />
-    ) : (
-      <IconExpand />
-    );
```

Comment:
unrelated changes?

Reply:
oh no, it is part of the [ticket](https://github.com/getsentry/sentry/issues/72846) . It is a very small change though and if you have strong feelings, I can create a new PR for it.